lib/random-schedule.js

```diff
@@ -0,0 +1,74 @@
+const Random = require('random-js')
+const random = new Random(Random.engines.browserCrypto)
+const Scheduler = require('azure-arm-scheduler')
+const randString = require('crypto-random-string')
+
+exports.calcSeconds = function (timestamp) {
+  var stamp = timestamp.split(':')
+  return 3600 * stamp[0] + 60 * stamp[1]
+}
+
+exports.calcRandomSecond = function (startTime, endTime) {
+  return random.integer(startTime, endTime)
+}
+
+exports.dayShortToLong = function (inputDays) {
+  var days = []
```

Comment:
let's try to do this validation as part of the cli tool arg builder option: https://github.com/yargs/yargs/blob/master/docs/advanced.md#providing-a-command-module